### ISSUE TITLE
ui5-date-picker: inaccurate accessibility attributes in relation to the combobox role removal

### ISSUE THREAD

**Issue by unazko** [SAP/ui5-webcomponents#4865]
### **Bug Description**
Due to redundant screen reader announcements caused by the <code>combobox</code> role the concept of the ui5-date-picker
was changed regarding the accessibility. Currently the ui5-date-picker input field isn't considered a <code>combobox</code>,
but an input field with a value help icon instead. As a result the <code>aria-expanded</code> attribute isn't conceptually correct any longer. 

### **Expected Behavior**
The <code>aria-expanded</code> attribute should be removed from the ui5-date-picker input element.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/master/playground/components/DatePicker/
2. Inspect the first ui5-date-picker instances input field.

### **Context**
- UI5 Web Components version: {latest}
- OS/Platform: {windows 10}
- Browser: {Chrome}
- Affected components: {ui5-date-picker, ui5-daterange-picker, ui5-datetime-picker}

### **Log Output / Stack Trace / Screenshots**
![2022-03-08_7-04-38](https://user-images.githubusercontent.com/15860295/157170264-2af13adf-dc07-4531-be0e-43a5a4247ff1.png)

### **Priority**
- [ ] Low
- [ ] Medium
- [X] High
- [ ] Very High

### **Stakeholder Info**

- Organization: {Success Factors}
- Business impact: {Blocked update to ui5-webcomponents version 1.2.1}

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

This issue is regarding the ui5-date-picker, ui5-daterange-picker and ui5-datetime-picker components, which are under our ownership.

Best Regards,
Boyan Rakilovski
